Cargo.toml

```diff
@@ -32,10 +32,10 @@ futures-core = "0.3.30"
 hex = "0.4.3"
 jsonrpsee = { version = "0.22", features = ["macros", "server", "client-core"] }
 log = "0.4"
-openmls = { git = "https://github.com/xmtp/openmls", rev = "0239b96" }
-openmls_basic_credential = { git = "https://github.com/xmtp/openmls", rev = "0239b96" }
-openmls_rust_crypto = { git = "https://github.com/xmtp/openmls", rev = "0239b96" }
-openmls_traits = { git = "https://github.com/xmtp/openmls", rev = "0239b96" }
+openmls = { git = "https://github.com/xmtp/openmls", rev = "9f3cad8c6c434f4860bd4f120451558e8d6293ce" }
```

Comment:
Adding a reminder that we're still pointing to a commit on the `nm/update-group-membership-action` branch in `xmtp/openmls`. Is the plan to merge that to `xmtp/openmls` `main` ?